docs/hybrid-slurm-cluster/hybrid-slurm.md

```diff
@@ -0,0 +1,423 @@
+An example showing how to do this with each of the default mount paths is
+provided below:
+
+```yaml
+- id: slurm-controller
+  source: community/modules/scheduler/schedmd-slurm-gcp-v5-hybrid
+  use:
+  - debug-partition
+  - compute-partition
+  settings:
+    output_dir: ./hybrid
+    slurm_bin_dir: /usr/local/bin
+    slurm_control_host: $(vars.static_controller_hostname)
+    install_dir: /etc/slurm/hybrid
+    network_storage:
+    - server_ip: cluster-controller-0 # IP or hostname of the slurm controller
+      remote_mount: /etc/slurm
+      local_mount: /usr/local/etc/slurm
+      fs_type: nfs
+      mount_options: ""
+    - server_ip: cluster-controller-0 # IP or hostname of the slurm controller
+      remote_mount: /etc/munge-install
+      local_mount: /etc/munge
+      fs_type: nfs
+      mount_options: ""
+    - server_ip: cluster-controller-0 # IP or hostname of the slurm controller
+      remote_mount: /apps
+      local_mount: /opt/apps
+      fs_type: nfs
+      mount_options: ""
+    - server_ip: cluster-controller-0 # IP or hostname of the slurm controller
+      remote_mount: /home/users
+      local_mount: /home
+      fs_type: nfs
+      mount_options: ""
+```
+
+[inputns]: ../../community/modules/scheduler/schedmd-slurm-gcp-v5-hybrid/README.md#input\_network\_storage
+
+#### Mount From Another Source
+
+If any of the default mount points exist in another network storage solution,
+such as an NFS /home filesystem, a `network_storage` entry can be added similar
+to [Create a New Network Storage Entry](#create-a-new-network-storage-entry),
+but with a different `server_ip`, as well as other customized options to suit
+your needs.
+
+### On Premise Partitions
+
+There are a couple of things to keep in mind when adding the hybrid
+configuration when it comes to the existing local partitions in your Slurm
+cluster.
+
+First of all, you will now be adding new partitions, the cloud partitions, when
+including the configuration produced by the hybrid module. If your current
+partitions are setting default values that apply broadly, they will also apply
+to the new cloud partitions.
+
+A common issue is when `Nodes=ALL` is set in a local partition definition. This
+will create duplicated partitions with the name of the local partition for all
+of the cloud partitions. To avoid this, set `Nodes` equal to the actual node
+names of the local partition.
+
+Another point to consider is whether your on premise nodes are using the Slurm
+[power saving][powersaving] functionality as described in
+[Power Saving Operations](#power-saving-operations) section. If they are, it will
+conflict with the mechanism used for the hybrid deployment. Supporting two
+distinct mechanisms for power saving operations is not supported, therefore the
+hybrid module will not support your use-case out of the box. It may be possible
+to customize the scripts to allow this, but it is not tested or supported by the
+HPC Toolkit.
```

Comment:
nit: you basically cover power saving twice. I would condense into a single section (probably above) and then here just say, "if you are using power saving see this section."

Reply:
I pretty much entirely cut this part out. When I tried combining them, I found it was almost entirely redundant, and the parts that weren't didn't seem necessary.